docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -177,4 +187,525 @@ movieUpdate(where: {
         title
     }
 }
+----
+
+
+== Subscribing to relationship events
+
+When subscribing to relationship events, the `where` argument still allows for specifying filters on the top-level properties of the targeted nodes, and also supports specifying filters on the relationship properties (`edge`) and on the top-level properties (`node`) of the nodes at the other end of the relationship. This is done by using the operators described above, and the usage is very similar to the one in xref:subscriptions/filtering.adoc#node-events-usage[Subscribing to node events].
+
+The relationship-related filtering logic is even more powerful, as filters can also express the expected relationship field, or the expected concrete type at the other end of the relationship when having relationships to Abastract Types. Note that these fields are combined in the resulting filter with a xref:subscriptions/filtering.adoc#filter-logical-or[logical `OR`].
+
+Only events matching these properties, conditions and type will be returned to this subscription. Note that these fields are combined in the resulting filter with a xref:subscriptions/filtering.adoc#filter-logical-and[logical `AND`].
+
+Considering the following type definitions:
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    genre: String
+    actors: [Actor!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+}
+interface ActedIn @relationshipProperties {
+    screenTime: Int!
+}
+type Actor {
+    name: String
+}
+----
+
+The format of the `where` argument is:
+[source, graphql, indent=0]
+----
+{
+    movie: {
+        # top-level properties of the node targeted for the subscription operation, supports operators
+        title_IN: []
+    },
+    createdRelationship: {
+        actors: { # field name corresponding to a relationship in the type definition of the node targeted for the subscription operation
+            edge: {
+                 # properties of the relationship, supports operators
+                screenTime_GT: 10,
+            },
+            node: {
+                # top-level properties of the node on the other end of the relationship, supports operators
+                name_STARTS_WITH: ""
+            }
+        }
+    }
+}
+----
+
+Below are some example of how filtering can be applied when creating a subscription to relationship events.
+
+=== Create Relationship
+The following example filters the subscriptions to newly created relationships that are connecting a `Movie` from genres other than "Drama", to an `Actor` with a screen time bigger than 10:
+
+[source, graphql, indent=0]
+----
+movieRelationshipCreated(where: { movie: { genre_NOT: "Drama" }, createdRelationship: { actors: { edge: { screenTime_GT: 10 } } } }) {
+    movie {
+        title
+    }
+    createdRelationship {
+        actors { 
+            screenTime
+            node { 
+                name
+            }
+        }
+    }
+}
```

Comment:
All of the query examples on this page are not actually valid because they're missing the `subscription` wrapper - this will need applying to all and reformatting.

```suggestion
subscription {
movieRelationshipCreated(where: { movie: { genre_NOT: "Drama" }, createdRelationship: { actors: { edge: { screenTime_GT: 10 } } } }) {
    movie {
        title
    }
    createdRelationship {
        actors { 
            screenTime
            node { 
                name
            }
        }
    }
}
}
```